docs/connectors/mysql.asciidoc

```diff
@@ -917,6 +917,12 @@ Columns that store strings are defined in MySQL with a character set and collati
 
 Other data type mappings are described in the following sections.
 
+If present, a column's default value will be propagated to the corresponding field's Kafka Connect schema.
```

Comment:
@gunnarmorling IIRC this is true also for cases where default value is set to all zeroes.